pkg/controller/certificates/trigger/trigger_controller.go

```diff
@@ -199,20 +199,37 @@ func (c *controller) ProcessItem(ctx context.Context, key string) error {
 	return nil
 }
 
-// shouldBackoffReissuingOnFailure tells us if we should back off from
-// reissuing the certificate and for how much time.
-func shouldBackoffReissuingOnFailure(log logr.Logger, c clock.Clock, crt *cmapi.Certificate) (backoff bool, delay time.Duration) {
+// shouldBackoffReissuingOnFailure tells us if this certificate's
+// re-issuance should be backed off and for how much time.
```

Comment:
The top hits for https://www.google.com/search?q=%22should+be+backed+off%22 are not pretty.

```suggestion
// shouldBackoffReissuingOnFailure tells us if we should back off
// re-issuing for this certificate and for how much time.
```